src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -2,27 +2,84 @@
 
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.junit.MatcherAssert.assertThat;
+import static org.joda.time.DateTimeConstants.APRIL;
 
 import java.net.MalformedURLException;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
 
+import org.folio.circulation.domain.policy.Period;
 import org.folio.circulation.support.http.client.IndividualResource;
 import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
 import org.junit.Test;
 
 import api.support.APITests;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.OverrideRenewalByBarcodeRequestBuilder;
+import api.support.builders.RenewByBarcodeRequestBuilder;
+import api.support.builders.RenewByIdRequestBuilder;
 import api.support.builders.RequestBuilder;
 import api.support.http.InventoryItemResource;
 
 public class RequestsAPILoanRenewalTests extends APITests {
+
+  @Test
+  public void forbidRenewalLoanByBarcode_firstRequestIsRecall() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
+    UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    requestsClient.create(new RequestBuilder()
+      .recall()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(pickupServicePointId)
+      .by(usersFixture.charlotte()));
+
+    Response response = renewByBarcodeClient.attemptCreate(
```

Comment:
Why is this using a resource client to perform a renewal rather than the renewal methods in the `LoansFixture`?

Were they insufficient in some way?

Reply:
Probably I didn't figure it out. I will change it.

Reply:
Done